docs/reference/flow/01-diagrams.md

```diff
@@ -60,12 +123,21 @@ di1 -- "now <= st" --> res_01
 di1 -- "now > st" --> res_rca
 ```
 
-### Uncovered Debt
+### Total Debt
+
+```mermaid
+flowchart TD
+rca([Total Debt])
+di0{ }
+res_00([sd ])
+res_01(["sd + od"])
 
-**Notes:** A non-zero uncovered debt implies:
+rca --> di0
+di0 -- "PAUSED" --> res_00
+di0 -- "STREAMING" --> res_01
```

Comment:
since it is the only diagram where we split them between statuses, i would suggest to use `rps > 0` and  `rps == 0`

Reply:
https://github.com/sablier-labs/docs/pull/200/commits/00d1bc7b034becc68fa9051fda9c09fbbc1ac519